### PR TITLE
New Chrome Dev Tools Cheat Sheet

### PR DESCRIPTION
**Pull request by vikashvverma**
IA Page: https://duck.co/ia/view/chrome_dev_tools_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/chrome_dev_tools.json [hunk 1]
```diff
@@ -0,0 +1,281 @@
+{
```
Comment (jdorweiler) [inline share/goodie/cheat_sheets/json/chrome_dev_tools.json]: can you rename this to chrome-dev-tools.json

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/chrome_dev_tools.json]: @jdorweiler Done

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=chrome dev tools cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chrome_dev_tools_cheat_sheet_mobile.png)

**Comment by jdorweiler** [comment]
:+1: looks great!  Sorry for the delay getting it merged @vikashvverma

**Comment by daxtheduck** [comment]
@vikashvverma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**